packages/react-native-worklets/plugin-oxc/src/worklet_pass.rs

```
@@ -0,0 +1,1452 @@
+                            &[]
+                        };
+                    for &i in arg_indices {
+                        if let Some(arg) = call.arguments.get_mut(i) {
+                            autoworkletize_function_arg(
+                                arg, ctx, state, scoping, builder, allocator, filename,
+                            );
+                        }
+                    }
+                    if is_object_hook_at_arg0(name) {
+                        if let Some(arg0) = call.arguments.get_mut(0) {
+                            if let Argument::ObjectExpression(obj) = arg0 {
```

Clippy (1.97.1)

[warning] this `if let` can be collapsed into the outer `if let`

(warning)